FlowCryptUI/Cell Nodes/ThreadDetailWebNode.swift

```diff
@@ -11,20 +11,58 @@ import AsyncDisplayKit
 public final class ThreadDetailWebNode: CellNode {
     public struct Input {
         let message: String?
+        let quote: String?
         let index: Int
 
-        public init(message: String?, index: Int) {
+        public init(message: String?, quote: String?, index: Int) {
             self.message = message
+            self.quote = quote
             self.index = index
         }
     }
 
     private let input: ThreadDetailWebNode.Input
 
-    private lazy var webViewNode: CustomWebViewNode = {
+    private lazy var messageNode: CustomWebViewNode = {
         let node = CustomWebViewNode()
         node.setAccessibilityIdentifier(accessibilityIdentifier: "aid-message-\(input.index)")
-        node.setHtml("""
+        node.setHtml(getFormattedHtml(html: input.message))
+        node.style.flexGrow = 1.0
+        return node
+    }()
+
+    private lazy var quoteNode: CustomWebViewNode = {
+        let node = CustomWebViewNode()
+        node.style.flexGrow = 1.0
+        return node
+    }()
+
+    private var shouldShowQuote = false
+
+    private lazy var toggleQuoteButtonNode: ASButtonNode = {
+        let configuration = UIImage.SymbolConfiguration(pointSize: 16, weight: .ultraLight)
+        let image = UIImage(systemName: "ellipsis", withConfiguration: configuration)
+        let button = ASButtonNode()
+        button.cornerRadius = 4
+        button.borderColor = UIColor.main.cgColor
+        button.borderWidth = 1
+        button.accessibilityIdentifier = "aid-message-\(input.index)-quote-toggle"
+        button.setImage(image, for: .normal)
+        button.contentEdgeInsets = .side(4)
+        button.imageNode.imageModificationBlock = ASImageNodeTintColorModificationBlock(.main)
+        button.addTarget(self, action: #selector(onToggleQuoteButtonTap), forControlEvents: .touchUpInside)
+        return button
+    }()
```

Comment:
we have the same button in `MessageTextSubjectNode`, let's extract it to separate class which will be used in `MessageTextSubjectNode` and `ThreadDetailWebNode`